src/OrchardCore.Modules/OrchardCore.Localization/Models/LocalizationSettings.cs

```diff
@@ -1,31 +1,36 @@
 using System.Globalization;
 
-namespace OrchardCore.Localization.Models
+namespace OrchardCore.Localization.Models;
```

Comment:
Please don't update existing files with this because it's breaking the diff in the PR and we can't see where the differences are. Only do it for new files.